examples/partial_weight_exchange_example/client.py

```diff
@@ -40,7 +43,7 @@ def evaluate(self, parameters: NDArrays, config: Config) -> Tuple[float, int, Di
         if not self.initialized:
             self.setup_client(config)
 
-        self.set_parameters(parameters, config)
+        self.set_parameters(parameters, config, fitting_round=False)
         testing = self.narrow_config_type(config, "testing", bool)
         num_classes = num_classes = self.narrow_config_type(config, "num_classes", int)
```

Comment:
I think we have an extra num_classes here.

Reply:
Yes thanks for spotting it!